app/config/logging-cfg-local.yml

```diff
@@ -42,6 +42,7 @@ filters:
     exclude_keys:
       - request.headers.Authorization
       - request.headers.Proxy-Authorization
+      - request.headers.Cookie
```

Comment:
This needs also to be  added to other logging configuration

Reply:
done